packages/arb-validator/rollupvalidator/txtracker.go

```diff
@@ -29,72 +37,90 @@ import (
 	"github.com/offchainlabs/arbitrum/packages/arb-util/hashing"
 	"github.com/offchainlabs/arbitrum/packages/arb-util/value"
 	"github.com/offchainlabs/arbitrum/packages/arb-validator-core/evm"
-	"github.com/offchainlabs/arbitrum/packages/arb-validator/rollup"
 )
 
-type validatorRequest interface {
-}
-
-type assertionCountRequest struct {
-	resultChan chan<- int
-}
-
-type outputMsgRequest struct {
-	assertionNodeHash common.Hash
-	msgIndex          int64
-	resultChan        chan<- value.Value
+type logsInfo struct {
+	Logs    []evm.Log
+	TxIndex uint64
+	TxHash  common.Hash
 }
 
-type txRequest struct {
-	txHash     common.Hash
-	resultChan chan<- txInfo
+type nodeInfo struct {
+	TxLogs            []logsInfo
```

Comment:
Suggest adding comments to explain the difference between TxLogs and OutLogs, and which things are covered by LogsAccHashes and LogsValHashes and TransactionHashes.

Reply:
Updated variable names and added comments with further explanation